harvard-lts/etd_alma_monitor_service
# Etd 282 3

**Add proquest id to tracing**
* * *

**JIRA Ticket**: [(link)](https://at-harvard.atlassian.net/browse/ETD-282)

# What does this Pull Request do?
This pr adds pqid as an attribute, which displays it and makes it searchable in alma monitor
# How should this be tested?

- Deploy to dev (done)
- run the alma monitor test: https://ltsds-cloud-dev-1.lib.harvard.edu:10610/alma_monitor_service
- go to dev tracing ui and search for identifier=30522803, it should be found in alma monitor tracing: http://ltsds-cloud-dev-1.lib.harvard.edu:16686/

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
Tag (@ mention) interested parties
